### PR TITLE
Adding two small deep links

### PR DESCRIPTION
**Pull request by sierisimo**

### CODE REVIEW

**Comment by iddar** [comment]
nel no me gusta

File: Android/Nothing/app/build.gradle.kts [hunk 1]
```diff
@@ -42,6 +53,13 @@ dependencies {
     implementation("com.google.firebase:firebase-messaging:21.0.1")
 
     implementation("com.github.jetradarmobile:android-snowfall:1.2.0")
+
+
```
Comment (iddar) [inline Android/Nothing/app/build.gradle.kts:57]: aqui hay un doble espacio

Comment (sierisimo) [reply Android/Nothing/app/build.gradle.kts:57]: chinga, debia ser triple...

**Review by gorrotowi — APPROVED** [review]
LGTM

REMINDER: Use canary version until new gradle plugin become stable or not, keep using whatever you orto says

**Pull request merged**